app/scripts/modules/core/src/notification/selector/types/githubstatus/GithubNotificationType.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { FormikFormField, TextInput, Validators } from 'core/presentation';
+import { INotificationTypeCustomConfig } from 'core/domain';
+
+export class GithubNotificationType extends React.Component<INotificationTypeCustomConfig> {
+  public render() {
+    const { fieldName } = this.props;
+    return (
+      <>
+        <div className="sp-margin-m-bottom">
+          <div className={'form-group'}>
```

Comment:
Another convention we try to follow is not using braces to wrap string literals in JSX. Think there are a few other places in this file we can make this change as well.
```suggestion
          <div className="form-group">
```